src/components/ButtonInfo.js

```diff
@@ -0,0 +1,37 @@
+function Buttons() {
+  const ButtonData = [
+    [
+      { name: 'AC' },
+      { name: '+/-' },
+      { name: '%' },
+      { className: 'orange', name: '/' },
```

Comment:
- [ ] The calculator can't divide.  All divisions result in a 0.

## Explantation and hints

On `operate.js`, the wrong symbol is being used for divisions:

![image](https://user-images.githubusercontent.com/57421823/158829901-824ef22b-79b2-4dbc-9cb4-99e52f5c2716.png)

Just change it for the correct symbol `/`:

![image](https://user-images.githubusercontent.com/57421823/158830081-6b90e25f-474b-44e7-bf16-625958a7c925.png)

Now, it should work! Go ahead and correct the bug now that you know where it is located 💪